kubernetes-regression-tests/src/test/resources/scripts/prepare-test-environment.sh

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
What exactly is this? Why do we need it here?

Reply:
Sorry, I forgot removing it. Actually, i had set up these tests for Travis CI, which used this script. I would remove it in next patch update.